package.json

```diff
@@ -136,6 +136,7 @@
     "ws": "^6.1.4"
   },
   "devDependencies": {
+    "@react-native-community/eslint-config": "*",
```

Comment:
Actually, this is wrong. Due to how our repository is set up right now (to work with our internal codebase), this only has to be declared in `react-native/repo-config/package.json` (and it is already present there).